### PR TITLE
108 admin login back

### PR DESCRIPTION
**Pull request by abdou059**

### CODE REVIEW

**Review by husamkamal — CHANGES_REQUESTED** [review]
why you push all admin login front and back together there is anther pull request that have front

**Review by LinaYahya — CHANGES_REQUESTED** [review]
delete unrelated changes, So I can review the code,
Also please take care of commit messages as it doesn't make sense to have three commits with the same message

File: server/src/services/auth.ts [hunk 1]
```diff
@@ -32,6 +32,16 @@ const signupUser = async ({
   return userInfo.id;
 };
 
+const checkAdmin = async ({ username }) => {
+  const getAdmin:any = await Admin.findOne({
```
Comment (LinaYahya) [inline server/src/services/auth.ts]:
```suggestion
  const getAdmin:any = await Admin.findOne({ 
```
remove any from here, and it's not a function to have an action name

**Review by LinaYahya — CHANGES_REQUESTED** [review]

File: server/src/services/auth.ts [hunk 2]
```diff
@@ -32,6 +32,16 @@ const signupUser = async ({
   return userInfo.id;
 };
 
+const checkAdmin = async ({ username }) => {
+  const findAdmin:any = await Admin.findOne({
```
Comment (LinaYahya) [inline server/src/services/auth.ts]:
```suggestion
  const admin = await Admin.findOne({
```

File: server/src/controllers/auth.ts [hunk 3]
```diff
@@ -70,8 +71,33 @@ const userController = async (req:Request) => {
   return { status: 401 };
 };
 
+const AdminLogin = async (req:Request) => {
```
Comment (LinaYahya) [inline server/src/controllers/auth.ts]:
```suggestion
const loginAdmin = async (req:Request) => {
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by mai-mohammed — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by husamkamal — APPROVED** [review]

**Review by tariqabusada — APPROVED** [review]

**Pull request merged**